### PR TITLE
Compute module for 2019-03-01-hybrid

### PR DESCRIPTION
**Pull request by viananth**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by bganapa** [comment]
https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign-esrp/120/ - ps-sign

File: src/StackAdmin/Compute/Commands.Compute.Test/Common/ComputeTestController.cs [hunk 1]
```diff
@@ -60,6 +61,9 @@ public sealed class ComputeTestController : RMTestBase
         public Subscriptions.SubscriptionClient SubscriptionClient { get; private set; }
 
         public GalleryClient GalleryClient { get; private set; }
+
```
Comment (bganapa) [inline src/StackAdmin/Compute/Commands.Compute.Test/Common/ComputeTestController.cs]: can you pls remove the commented code in this file?

File: src/StackAdmin/Compute/Commands.Compute.Test/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -32,6 +32,6 @@
 
 [assembly: Guid("28fe3fc0-6fdf-4f01-8a82-d7b4a834ddd2")]
 
-[assembly: AssemblyVersion("4.0.3")]
-[assembly: AssemblyFileVersion("4.0.3")]
+[assembly: AssemblyVersion("4.0.1")]
```
Comment (bganapa) [inline src/StackAdmin/Compute/Commands.Compute.Test/Properties/AssemblyInfo.cs]: can you pls change this to 4.6.1

File: src/StackAdmin/Compute/documentation/upcoming-breaking-changes.md [hunk 3]
```diff
@@ -1,28 +1,17 @@
-<!--
-    Please leave this section at the top of the breaking change documentation.
+# Upcoming Breaking Changes
```
Comment (bganapa) [inline src/StackAdmin/Compute/documentation/upcoming-breaking-changes.md]: pls revert the changes in this file

**Comment by maddieclayton** [comment]
@bganapa Is this ready for us to review and merge?

**Comment by bganapa** [comment]
Thanks for the pin @maddieclayton . This is ready for your review

File: src/StackAdmin/Compute/Commands.Compute/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -25,8 +25,8 @@
 [assembly: ComVisible(false)]
 [assembly: CLSCompliant(false)]
 [assembly: Guid("91792853-487B-4DC2-BE6C-DD09A0A1BC10")]
-[assembly: AssemblyVersion("4.0.3")]
-[assembly: AssemblyFileVersion("4.0.3")]
+[assembly: AssemblyVersion("4.6.0")]
```
Comment (maddieclayton) [inline src/StackAdmin/Compute/Commands.Compute/Properties/AssemblyInfo.cs]: This should be 4.6.1 according to the psd1

File: src/StackAdmin/Compute/Commands.Compute/Commands.Compute.csproj [hunk 5]
```diff
@@ -54,13 +54,16 @@
...
     <Reference Include="Microsoft.Azure.Management.Compute">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Compute.16.3.0\lib\net452\Microsoft.Azure.Management.Compute.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Compute.17.5.0\lib\net452\Microsoft.Azure.Management.Compute.dll</HintPath>
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.Azure.Management.KeyVault, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.KeyVault.2.3.0-preview\lib\net452\Microsoft.Azure.Management.KeyVault.dll</HintPath>
       <Private>True</Private>
     </Reference>
+<!--    <Reference Include="Microsoft.Azure.Management.Storage">
```
Comment (maddieclayton) [inline src/StackAdmin/Compute/Commands.Compute/Commands.Compute.csproj]: Remove this commented out code
